sentry-spring/src/main/java/io/sentry/spring/graphql/SentryGraphqlConfiguration.java

```diff
@@ -14,31 +17,38 @@
 @Open
 public class SentryGraphqlConfiguration {
 
-  @Bean
+  @Bean(name = "sentryInstrumentation")
+  @ConditionalOnMissingBean(name = "sentryInstrumentation")
   @ConditionalOnWebApplication(type = ConditionalOnWebApplication.Type.SERVLET)
-  public GraphQlSourceBuilderCustomizer sourceBuilderCustomizerWebmvc() {
+  public SentryInstrumentation sentryInstrumentationWebMvc(
+      final @NotNull ObjectProvider<SentryGraphqlInstrumentation.BeforeSpanCallback>
+              beforeSpanCallback) {
     SentryIntegrationPackageStorage.getInstance().addIntegration("Spring5GrahQLWebMVC");
-    return sourceBuilderCustomizer(false);
+    return createInstrumentation(beforeSpanCallback, false);
   }
 
-  @Bean
+  @Bean(name = "sentryInstrumentation")
+  @ConditionalOnMissingBean(name = "sentryInstrumentation")
```

Comment:
We should change this to not rely on the bean name but rather the type `SentryInstrumentation` regardless of name.